src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunReportPageTests.kt

```diff
@@ -175,4 +196,23 @@ class RunReportPageTests : SeleniumTest()
         changelogType.selectByValue("public")
         assertThat(changelogType.firstSelectedOption.getAttribute("value")).isEqualTo("public")
     }
+
+    private fun reportIsAvailable(reportName: String): Boolean
+    {
+        val typeahead = driver.findElement(By.id("report"))
+        val input = typeahead.findElement(By.tagName("input"))
+        input.clear()
+        input.sendKeys(reportName)
+
+        val listItem = typeahead.findElements(By.tagName("li"))[0]
+        val foundReportName = if (listItem.getAttribute("role") == "option")
+        {
+            listItem.findElements(By.tagName("span"))[0].getAttribute("innerHTML")
```

Comment:
For some reason, using `text` here seemed a bit flaky and sporadically gave results which didn't match the html - so `innerHTML` it is...